path/posix/separator.ts

```diff
@@ -1,5 +1,11 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 // This module is browser compatible.
 
+/**
+ * @deprecated (will be removed in 1.0.0) Use "separator" in https://deno.land/std@$STD_VERSION/path/posix/constants.ts instead.
```

Comment:
I don't think the IDE (deno lsp) can render `$STD_VERSION` part as the corresponding version number, and this url notation can be confusing. How about just omitting the urls in deprecation notes?

Reply:
Including URLs is fine if the `@$STD_VERSION` portion is omitted. The documentation should be rendered fine if we also wrap the URL in a `{@link <url>}` tag. E.g. the `win32` export in `path/mod.ts` renders correctly.